.changeset/spicy-numbers-deny.md

```diff
@@ -0,0 +1,20 @@
+---
+"@commercetools/composable-commerce-test-data": patch
+---
+
+Added new submodel to the ProductSelection model:
+
+- `ProductOfSelection`
+
+You can use it like this:
+
+```ts
+import {
+  ProductOfSelectionGraphql,
+} from '@commercetools/composable-commerce-test-data/product-selection';
```

Comment:
Also, should it be exposed at all from the index file? Cause it's not supposed to be used outside of the parent model, or is it?

Reply:
Yes they should be exposed. 
Remember consumers need the ability to populate the `productRefs` property and they should use the new models for that.